scripts/azureml-assets/azureml/assets/release/model_publish_utils/mlflow_model_utils.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import logging
+import os
+import azureml.evaluate.mlflow as mlflow
+from mlflow.models import ModelSignature
+from sqlalchemy import true
+from transformers import  AutoTokenizer, AutoConfig, pipeline
+from transformers import (
+    AutoModelForSequenceClassification,
+    AutoModelForMaskedLM,
+    AutoModelForTokenClassification,
+    AutoModelForQuestionAnswering,
+    AutoModelWithLMHead
+)
+
+class MLFlowModelUtils:
+
+    def __init__(self, name, task_name, model_dir):
+        self.name = name
```

Comment:
@wayliums @elesel Wondering if we are using type hints in this repo?

Reply:
We are. They're not required, but they certainly help.